gstvstaudioprocessor.cpp

```diff
@@ -17,6 +17,14 @@
  *
  */
 
+#include <sys/types.h>
+#include <sys/stat.h>
+#ifdef HAVE_UNISTD_H
```

Comment:
This needs a meson check, but as it would now never be #defined and it compiled for you I assume the header is not needed at all? :)

Reply:
Too much copy pasting, you are right :)